src/nvim/os/stdpaths.c

```diff
@@ -2,13 +2,16 @@
 #include <stdbool.h>
 #include <string.h>
 
+#include "klib/kvec.h"
 #include "nvim/ascii_defs.h"
 #include "nvim/fileio.h"
+#include "nvim/garray.h"
```

Comment:
This doesn't seem needed

Reply:
seems like before auto import forget to remove.